client/post-editor/editor-confirmation-sidebar/style.scss

```diff
@@ -93,8 +93,8 @@
 }
 
 .editor-confirmation-sidebar__action .button {
-	background: var( --color-success );
-	border-color: darken( $alert-green, 20% );
+	background: var( --color-accent );
+	border-color: var( --button-primary-border-color );
```

Comment:
It seems like we're rebuilding the primary button at this place. What do you think about removing those styles altogether and use `<Button primary ... />` instead?

Reply:
Currently, yes, it's intended to be the same as the Publish button is before the pre-publish confirmation, because I think that was suggested in the original issue. A change in colour would be nice though, because that's what currently happens. Would it be better making the colour of the button `color-primary` instead? Let me know if I misunderstood. :)

Reply:
Sorry, I wasn't clear. What I meant is getting rid of that `.editor-confirmation-sidebar__action .button` block altogether and make the `<Button />` a primary button [here](https://github.com/Automattic/wp-calypso/blob/master/client/post-editor/editor-confirmation-sidebar/index.jsx#L92) by adding the `primary` prop:

```jsx
<Button primary disabled={ disabled } onClick={ this.closeAndPublish }>
  { buttonLabel }
</Button>
```

Reply:
Should probably fix up https://github.com/Automattic/wp-calypso/blob/master/client/post-editor/editor-confirmation-sidebar/index.jsx#L132 too while we're at it

Reply:
Ah, gotcha, my bad. That's probably better, since looking at this again, there's some other bits of green too for the pre-Publish button. I've added some commits to address both, let me know your thoughts? :)